src/rdkafka_admin.c

```diff
@@ -839,6 +957,114 @@ rd_kafka_admin_worker (rd_kafka_t *rk, rd_kafka_q_t *rkq, rd_kafka_op_t *rko) {
 
 }
 
+static rd_kafka_op_res_t
+rd_kafka_admin_fanout_worker (rd_kafka_t *rk, rd_kafka_q_t *rkq, rd_kafka_op_t *rko);
+
+static rd_kafka_op_t *
+rd_kafka_admin_fanout_op_new (rd_kafka_t *rk,
+                              rd_kafka_op_type_t req_type,
+                              rd_kafka_event_type_t reply_event_type,
+                              const struct rd_kafka_admin_fanout_worker_cbs* cbs,
+                              const rd_kafka_AdminOptions_t *options,
+                              rd_kafka_queue_t *rkqu) {
+        rd_kafka_op_t *rko;
+
+        rd_assert(rk);
```

Comment:
This assert is not necessary, there is very little risk that someone calls this with a NULL rk.  The rkqu is okay though.